virtualbox/vbox-adapter-check.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
```

Comment:
missing copyright header

Reply:
sorry, there is one single file with such header, the linter that @mr-tz contributed: [scripts/test/lint.py](https://github.com/mandiant/VM-Packages/blob/921ab89f37a2513fa77571f695415b2160970b03/scripts/test/lint.py#L8)

Reply:
adding copyright headers to all files and a LICENSE.txt would bring the project closer in line to Google recommendations, but should be its own effort, not this PR.